src/app.ts

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable camelcase */
+import { Bytes, KeyType, PrivateKey, PublicKey } from '@greymass/eosio';
+import randomBytes from 'randombytes';
+import { ES256KSigner, createJWT } from 'did-jwt';
+import * as jose from 'jose';
+import { PublicKey as PublicKeyCon } from 'eosjs/dist/eosjs-key-conversions';
+function generateRandomKeyPair(): { privateKey: PrivateKey; publicKey: PublicKey } {
+    const bytes = randomBytes(32);
+    const privateKey = new PrivateKey(KeyType.K1, new Bytes(bytes));
+    const publicKey = privateKey.toPublic();
+    return { privateKey, publicKey };
+}
+
+async function onPressLogin(redirect: string): Promise<string> {
+    const { privateKey, publicKey } = generateRandomKeyPair();
+    const payload = {
+        number: Math.floor(Math.random() * 100),
```

Comment:
lets use `randomString` or `randomBytes` here for cryptographically secure random number